packages/project-integrity/src/fiori-project/index.ts

```diff
@@ -0,0 +1,118 @@
+import { existsSync } from 'fs';
+import { join } from 'path';
+import { getCapCustomPaths } from '@sap-ux/project-access';
+import {
+    checkProjectIntegrity,
+    disableProjectIntegrity,
+    enableProjectIntegrity,
+    initProject,
+    isProjectIntegrityEnabled,
+    updateProjectIntegrity
+} from '../integrity';
+import type { CheckIntegrityResult, Content } from '../types';
+
+export const fioriIntegrityDataPath = join('.fiori-ai/ai-integrity.json');
+
+/**
+ * Get the list of files to protect the integrity of.
+ *
+ * @param projectRoot - root folder of the project
+ * @returns - list of file paths
+ */
+async function getFileList(projectRoot: string): Promise<string[]> {
+    const fileList: string[] = [];
+
+    const schemaCds = join(projectRoot, 'db/schema.cds');
```

Comment:
Afaik the folder and the actual name of the schema (and srv) file is not hard coded. It depends on the [build configuration](https://cap.cloud.sap/docs/guides/deployment/custom-builds#build-config). Do we really want to hard code this?

Reply:
Also, why is it an error it these files do not exist?

Reply:
@heimwege, this is the Fiori project case where we generate the CAP project using AI, so the project structure is ensured. This is also one reason for this integrity module, to ensure a certain project structure as it requires way more effort to support any kind of CAP project.